src/lib/Form/Type/FieldDefinition/FieldDefinitionType.php

```diff
@@ -26,25 +27,23 @@
  */
 class FieldDefinitionType extends AbstractType
 {
-    /**
-     * @var \EzSystems\EzPlatformAdminUi\FieldType\FieldTypeDefinitionFormMapperDispatcherInterface
-     */
+    /** @var \EzSystems\EzPlatformAdminUi\FieldType\FieldTypeDefinitionFormMapperDispatcherInterface */
     private $fieldTypeMapperDispatcher;
 
-    /**
-     * @var FieldTypeService
-     */
+    /** @var FieldTypeService */
```

Comment:
Nitpick: 
```suggestion
    /** @var \eZ\Publish\API\Repository\FieldTypeService */
```
(since you're aligning with code style ;))